plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -157,9 +169,9 @@
       // Border:
       if (borderSize > 0) {
         context.beginPath();
-        context.fillStyle = settings('nodeBorderColor') === 'node' ?
-          (color || defaultNodeColor) :
-          settings('defaultNodeBorderColor');
+        context.fillStyle = settings('nodeBorderColor') === 'node'
+          ? borderColor
```

Comment:
but this would be wrong because when `nodeBorderColor` == 'node' the colour of the border should be what's indicated in the node i.e. `node.border_color` (not `node.color`), which if not provided should default to `node.color`.  so:

```
borderColor = settings('nodeBorderColor') === 'node' 
  ? (node.border_color || color || defaultNodeColor)
  : (o.borderColor || node.border_color || settings('defaultNodeBorderColor') || defaultNodeColor),
```

but I prefer my proposed alternative because the semantics are clearer.  if I set the *nodeBorderColor* to *default* then there's really only 1 colour that I should use: `settings('defaultNodeBorderColor')`- and it better be provided, so there are no defaults.  so something like this would be better:
```
borderColor = settings('nodeBorderColor') === 'default'
          ? settings('defaultNodeBorderColor')
          : (node.border_color || o.borderColor ||  defaultNodeColor),
```

notice I've given higher priority to the node border colour because, if provided, that should override the passed-in options, and if neither is provided we use the default node colour.